main.go

```diff
@@ -79,11 +79,669 @@ var (
 
 	masterMetricsLabels = []string{"host"}
 
+	// http://mesos.apache.org/documentation/latest/monitoring/
 	masterMetrics = []snapshotMetric{
 		snapshotMetric{
 			desc: prometheus.NewDesc(
+				"mesos_master_cpus_fraction",
+				"Fraction of allocated CPUs.",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/cpus_percent",
```

Comment:
Does this need to be scaled, or am I right in reading the docs that Mesos has confused percent and ratio?